packages/perseus-editor/src/__stories__/flags-for-api-options.ts

```diff
@@ -19,6 +19,7 @@ export const flags = {
         // Start coords UI flags
         // TODO(LEMS-2228): Remove flags once this is fully released
         "start-coords-ui-phase-1": true,
+        "start-coords-ui-phase-2": true,
```

Comment:
I can remove this and all the extra flag code below if we want to release sinusoid as part of phase 1.